Trace/TraceModel/TraceModel+Span.swift

```diff
@@ -301,6 +306,31 @@ extension TraceModel {
             
             return copy
         }
+        
+        // MARK: - Span validator
+        
+        /// :nodoc:
+        @discardableResult
+        internal func validate() -> Bool {
+            guard let strongEnd = end else {
+                return false
+            }
+            
+            var valid = true
+            if start.seconds > strongEnd.seconds {
+                valid = false
+            } else if start.seconds == strongEnd.seconds {
+                if start.nanos > strongEnd.nanos || start.nanos == strongEnd.nanos {
```

Comment:
Can you use `>=` as it's easier to read? This way you would only need one check

Reply:
Thanks updated.